sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -409,15 +410,16 @@ async def receive_messages(self, max_message_count=None, max_wait_time=None):
             require_timeout=True
         )
 
-    async def receive_deferred_messages(self, sequence_numbers):
-        # type: (Union[int, List[int]]) -> List[ReceivedMessage]
+    async def receive_deferred_messages(self, sequence_numbers, timeout=None):
```

Comment:
Same comment as above - timeout should only be a keyword-only parameter for all the async operations.

Reply:
updated, thanks